lib/clusterable/ht_item.rb

```diff
@@ -26,38 +26,57 @@ class HtItem
     ACCESSOR_ATTRS.each { |attr| attr_accessor attr }
     READER_ATTRS.each { |attr| attr_reader attr }
 
-    def self.db
-      Services.hathifiles_table
-    end
+    class << self
```

Comment:
I think I prefer having each class method defined as `def self.foo ...`  as opposed to being defined inside a `class << self` block. Just for readability.

Reply:
ah gotcha